Client/Frontend/ContentBlocker/ContentBlockerHelper+TabContentScript.swift

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import WebKit
+import Shared
+import Deferred
+
+@available(iOS 11, *)
+extension ContentBlockerHelper : TabContentScript {
```

Comment:
This would have been caught by CI. But there shouldnt be a space between `ContentBlockerHelper` and the semi-colon